src/ufoLib2/objects/font.py

```diff
@@ -188,6 +195,10 @@ def open(cls, path: PathLike, lazy: bool = True, validate: bool = True) -> "Font
             validate: If True, enable UFO data model validation during loading. If
                 False, load whatever is deserializable.
         """
+        if have_iondrive:
+            import ufoLib2.objects
+
+            return iondrive.load(ufoLib2.objects, str(path))
```

Comment:
how about we let caller pass a callable that takes the ufoLib2.objects namespace and the path and returns a Font object? That way we don't need to call out `iondrive` in particular, but the client can pass anything that quacks like a duck

Reply:
I'll do this, but it feels pointless. If the client's going to have to write `Font(whatever, loader=iondrive)` they may as well just write `iondrive.load(ufoLib2.objects, whatever)`.

Reply:
Then that might even be preferable. Fontmake could be made to use it if it exists.

Reply:
yes, I agree it's kinda pointless. Let's not touch ufoLib2, but move the `iondrive.load(ufoLib2.objects, path)` call upstream in fontmake